### DISCUSSION TITLE
Turning off the obnoxious tooltip popups

### DISCUSSION THREAD

**Discussion by fantasai** [community/community#44935] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

It seems like every time I move my mouse there's some new popup obscuring the page. Hovering over almost everything triggers a popup, and the issue tracker has therefore become incredibly obnoxious to use.

(It's even worse on narrow windows, because sometimes the popup is placed incorrectly and there's no way to actually click on the thing I'm trying to click on because I'm hovering over a popup that obscures it.)

Can there be a way to turn these off, please. I just want no popups, for nothing, ever. They just get in the way and make this tool that I unfortunately have to use every day unnecessarily frustrating.

**Comment by hpiwowar** [comment]
totally agree! very frustrating!

**Comment by aib** [comment]
This is so bad, I'm wondering whether it's specific to Firefox or my configuration. Who thought infinite-duration tooltips was a good idea?!

**Reply by cocoelacanth** [reply]
> This is so bad, I'm wondering whether it's specific to Firefox or my configuration. Who thought infinite-duration tooltips was a good idea?!

Looks like a Firefox bug. I tested this in Microsoft Edge and the tooltips disappear immediately when you move your mouse.

This is _beyond_ annoying. Like, this is actually impeding my ability to use GitHub properly, because the tooltips keep blocking parts of the page that I need to click on.

This is why we need more than one browser engine. Look what happens when Google gets the monopoly.

**Reply by ilyvion** [reply]
I've been using Firefox with GH for some time now, and this never-closing tooltip issue only started happening recently.

**Comment by hpiwowar** [comment]
Agreed, for me in Firefox too. It seems like a bug because the tips on an
Issue page don't go away even when you click outside them.

**Comment by JelsB** [comment]
It's indeed a Firefox bug. But an upgrade to version 109 fixed it for me.

**Reply by rehanvdm** [reply]
Worked for me!

**Reply by cocoelacanth** [reply]
I'm already on Firefox 109 and it still seems to be an issue.

**Reply by amalpotra** [reply]
Even on 110 beta, it exists.

**Reply by TurboGit** [reply]
> Worked for me!

It seems it works when you first start Firefox, then the bug is back again. You can also reload the page to fix the issue temporarily. Veyr very annoying issue!

**Comment by hpiwowar** [comment]
I'm on 109 and still a problem for me.  Had to use Chrome to get things
done.  Frustrating!
Heather

**Comment by rehanvdm** [comment]
Yeah, the upgrade worked for me in the beginning, but now it is back again. Closing the browser and opening seems to work? Or maybe it is just if the page was open a long time 🤷‍♂️ what an annoying intermittent problem.

**Reply by VisualBean** [reply]
Ive noticed other css changes, like the left-hand file-tree view in PR-code tab. So perhaps related to some general css changes

**Comment by darchitectlp** [comment]
This bug is infuriating. Any updates? Using Firefox 102.7.0esr for mac. Lol, also, trying to google this issue was difficult. How do you bug report the bug reporting site? Ha!

**Comment by koseki** [comment]
I've signed out and signed in again, then it seems to be fixed.
macOS, Firefox 109.

**Reply by hpiwowar** [reply]
Alas that didn't fix it for me.  Also macOS, Firefox 109.

**Comment by radar-blip** [comment]
👉🏻    #44630

**Comment by queenofcorgis** [comment]
Hi y'all 🤠  it looks like we have a fairly promising fix in the works - thread [here](https://github.com/orgs/community/discussions/44630#discussioncomment-4833394) for more information and updates.
